specifyweb/specify/migrations/0027_CO_children.py

```
@@ -1,8 +1,158 @@
+
+HIDDEN_FIELDS = [
+    "timestampcreated", "timestampmodified", "version", "createdbyagent", "modifiedbyagent"
+]
+
+
+def datamodel_type_to_schematype(datamodel_type: str) -> str:
+    return "".join(map(lambda type_part: type_part.lower().capitalize(), datamodel_type.split('-')))
+
+
+def camel_to_spaced_title_case(camel_case: str) -> str:
+    return re.sub(r"(?<!^)(?=[A-Z])", " ", camel_case).title()
```

Ruff (0.15.13)

[error] Undefined name `re`

(F821)